src/mixins/enum-mixin.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { EntityTypes } from '@/enums'
+
+/**
+ * Mixin that provides some useful enum-related utilities.
+ */
+@Component({})
+export default class EnumMixin extends Vue {
```

Comment:
This is part of the mixin from Filings UI. It is useful for changing types to strings.